update/common/scripts/5.4/cleanuntranslatablerelations.php

```diff
@@ -53,9 +53,16 @@
  */
 function parseRelationListIds( $relationListXml )
 {
+    if ( empty( trim( $relationListXml ) ) )
+        return array();
+
     $xml = simplexml_load_string( $relationListXml );
     $list = $xml->{'relation-list'};
 
+    if ( ! $list instanceof SimpleXMLElement ) {
```

Comment:
Good point, I find negating the `instanceof` operator strange imho ... 
It should read `( ! ( $var instanceof Class ) )`, but the instanceof has higher precedence hence no parenthesis needed; I just tend to read `!$var instanceof Class` the wrong way (negating first)...

As for legacy CS I don't know the exact rule atm, but a search shows both these forms:
```
if( !$user instanceof eZUser )
if ( !( $user instanceof eZUser ) )
```

So... I'll just remove the space.